general/server/plugins/add-ons.md

```diff
@@ -0,0 +1,86 @@
+---
+uid: server-plugins-Add-ons
+title: Add-ons
+---
+### Introduction
```

Comment:
```suggestion
# Add-ons
```
Other pages start with the title and have the description immediately under this header.